statsmodels/base/model.py

```diff
@@ -250,6 +250,33 @@ def hessian(self, params):
         """
         raise NotImplementedError
 
+    def _get_start_params(self, **kwargs):
+        """
+        Find reasonable starting points for parameter optimization.
+
+        Parameters
+        ----------
+        **kwargs : Keyword arguments passed to `fit`
+            Not used in base class implementation
+
+        Returns
+        -------
+        start_params : np.ndarray
+
+        Raises
+        ------
+        ValueError if start_params cannot be found.
+        """
+        if hasattr(self, 'start_params'):
+            start_params = self.start_params
```

Comment:
This line exists in LikelihoodModel.fit ATM.  If it is never hit, we should consider removing this usage.

Reply:
Is tt never hit in the existing tests, or unhittable?  I agree with @ChadFulton  that sv should be user settable, ideally.

Reply:
It can be hit by setting `model.start_params = foo` before calling `model.fit(...)`, but this can also be accomplished by passing `start_params=foo` to `model.fit`.

Reply:
attributes or methods like this are or were for subclassing.
e.g. a subclass of GenericLikelihoodModel can specify appropriate `start_params` without having to overwrite the `fit` method.

Reply:
This side-note can be put on the back-burner.  This PR leaves the logic unchanged, so GenericLikelihoodModel subclasses can keep doing whatever they want.